### PR TITLE
Products resource now implements all product APIs

### PR DESCRIPTION
**Pull request by rdjpalmer**

### CODE REVIEW

**Comment by iest** [comment]
:ship:

File: modules/resources/Products.js [hunk 1]
```diff
@@ -5,19 +5,46 @@ import methodTypes from '../methodTypes';
...
+        method: methodTypes.GET,
+        path: 'bundle',
+        queryParams: ['sku', 'available', 'page', 'per_page'],
+      }),
+      hardware: pactMethod({
+        method: methodTypes.GET,
+        path: 'hardware',
+        queryParams: ['sku', 'available', 'page', 'per_page'],
+      }),
       coffee: pactMethod({
```
Comment (iest) [inline modules/resources/Products.js]: Actually I don't like these names.

Comment (iest) [inline modules/resources/Products.js]: They should be verbs describing what's actually happening.

Comment (rdjpalmer) [inline modules/resources/Products.js]: `getBundle`, `getHardware`... additional of get seems unnecessary.

Comment (iest) [inline modules/resources/Products.js]: ???

They're methods, not resources. They should be named as such/

**Comment by iest** [comment]
:ship:

**Pull request merged**